README.adoc

```diff
@@ -196,3 +196,33 @@ $ make snapshot-rpm
 You might need to install missing dependencies. The build process itself probably takes quite some time.
 
 You're going to find `results/YYYYMMDD/snapshot-rpm.log` with logging everything from this makefile target.
+
+== Mass Rebuilds ==
+
+This repository uses GitHub Actions to periodically perform rebuilds of select
```

Comment:
```suggestion
This repository uses GitHub Actions to periodically perform rebuilds of selected
```

I'm not sure this is a typo because I'm no native speaker.

Reply:
I think it's grammatically correct, but I'll adopt your suggestion since it is more readable.